src/tslint-cli.ts

```diff
@@ -50,6 +50,9 @@ let processed = optimist
             alias: "exclude",
             describe: "exclude globs from path expansion",
         },
+        fix: {
+            describe: "Fixes some linting errors (may overwrite linted files)",
```

Comment:
"some" isn't great language -- can you just use the same description you have on L159